source/managing-teams/leading-engagements.html.md

```diff
@@ -0,0 +1,90 @@
+  and creating "gatekeepers" such as project managers. We do the opposite: we encourage the team to
+  interact with clients as much as possible, promoting public communication and direct interactions.
+  This helps the team learn more about the client's vision and industry, and feel like they're part
+  of the client's mission. 
+- **Allow people to take ownership.** As a manager, you might be tempted to own every single goal
+  and initiative your team commits to. This kind of micromanagement will alienate your team and lead
+  you to burnout. Instead, find ways to cleanly and responsibly delegate ownership to your team, and
+  limit yourself to supporting them. This will seem scary at first, but it will bring out the best
+  in your team, and it's the only sane way to scale your workload in the long term.
+- **Create space for learning.** In an ideal world, your team members will be learning something
+  new with every single initiative they undertake. While this may not always be possible, and boring
+  work will always be part of the job, you should continuously look for ways to create learning
+  opportunities for your team, by putting the right people on the right projects and creating safe
+  boundaries for experimentation and innovation. You can read more about this in
+  [Promoting Growth](/managing-teams/promoting-growth/).
+
+One aspect to keep in mind is that there's a fine balance between empowering your team and
+protecting them from external pressure and time wasting. The latter activity is commonly referred to
+as **being a shit umbrella.** Finding the right balance requires time and constant introspection,
+and it's possibly one of the most valuable and critical aspects of management work.
+
+## Supporting Nebulab
+
+Engineering Managers are in the perfect sweet spot between hands-on and executive work: they
+interact directly with teammates and clients, so they have first-hand understanding of how well our
+values, practices and policies translate into the real world; at the same time, they report to the
+Directors, so they also have the resources they need to ignite company-wide changes (e.g., through
+the [peer-level meeting](/managing-teams/peer-level-meeting/)).
+
+In addition to ensuring the project's and the team's success, you should always be on the lookout
+for ways to improve Nebulab as a whole. Are there are any practices in your team that you're finding
+particularly beneficial or detrimental? What could improve your team's life and happiness? How can
+we as a company better serve the needs of our clients?
+
+While not a part of your day-to-day work, these kind of introspective questions should be a core
+part of your job here. None of our policies are set in stone: we're continuously reinventing
+ourselves to become a better workplace for our teammates and a better consulting firm for our
+clients, and we can only do that with your feedback and suggestions.
```

Comment:
I suspect this might be a good place to also talk about loom project updates prepared by EMs at the all-hands.

Reply:
@mtylty agreed, although before we do that, I think we should probably alter the all-hands chapter slightly, so maybe let's wait for that?